### PR TITLE
DVCSMP-4708 Wallmote buttons not setting supported button values

### PR DESCRIPTION
**Pull request by greens**

### CODE REVIEW

File: devicetypes/smartthings/aeotec-wallmote.src/aeotec-wallmote.groovy [hunk 1]
```diff
@@ -149,6 +149,7 @@ def createChildDevices() {
             child = addChildDevice("Child Button", "${device.deviceNetworkId}:${i}", device.hubId,
                     [completedSetup: true, label: "${device.displayName} button ${i}",
                      isComponent: true, componentName: "button$i", componentLabel: "Button $i"])
+            child.sendEvent(name: "supportedButtonValues", value: ["pushed", "held"])
```
Comment (dkirker) [inline devicetypes/smartthings/aeotec-wallmote.src/aeotec-wallmote.groovy]: Don't forget to encode as JSON and `displayed: false`!

Comment (tpmanley) [inline devicetypes/smartthings/aeotec-wallmote.src/aeotec-wallmote.groovy]: Displayed: false?

**Comment by greens** [comment]
updated

**Review by dkirker — DISMISSED** [review]

File: devicetypes/smartthings/aeotec-wallmote.src/aeotec-wallmote.groovy [hunk 2]
```diff
@@ -51,8 +51,8 @@ def getNumberOfButtons() {
 def installed() {
     createChildDevices()
     sendEvent(name: "numberOfButtons", value: numberOfButtons)
```
Comment (tpmanley) [inline devicetypes/smartthings/aeotec-wallmote.src/aeotec-wallmote.groovy]: This should also be displayed: false

**Review by dkirker — APPROVED** [review]

**Pull request merged**